client/components/telecom/telephony/alias/liveCalls/eavesdrop/telecom-telephony-alias-configuration-liveCalls-eavesdrop.html

```diff
@@ -46,16 +46,16 @@
                        data-ng-pattern="/^\+?\d+$/"
                        autofocus
                        required />
-                <p class="alert alert-danger top-space-m16"
+                <p class="alert alert-danger mt-3"
                    role="alert"
                    data-ng-if="$ctrl.error"
                    data-ng-bind="$ctrl.error">
                 </p>
                 <button type="submit"
-                        class="btn btn-primary float-right top-space-m8"
+                        class="btn btn-primary float-right mt-2"
```

Comment:
`.float-right` is needed ?

`div.text-right` is easier to manager, or don't forget to put a `.clearfix`

Reply:
Do you use a right-floated button in this case on Web pages? There should be inconstancy on this part (tbc on phase 2...)